src/components/common/Questionnaire/ExpandedModal/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { Fragment, useCallback, useState } from 'react';
+import styles from './NetPromoter.module.css';
+import Question from './Question';
+import NextButton from './NextButton';
+import questionList from '../questionList';
+import AppreciationContent from './AppreciationContent';
+import { postUserFeedback } from 'actions/userFeedback';
+import { connect } from 'react-redux';
+
+const ExpandedModal = ({ handleToggleModalOpen, postUserFeedback }) => {
```

Comment:
`connect` 是舊版React的用法，現在用hook的話就不建議用 connect 的 HOC 寫法了。建議是：

```suggestion
import { useDispatch } from 'react-redux';

const ExpandedModal = ({ handleToggleModalOpen }) => {
  const dispatch = useDispatch()
  // ..... 
  const handleSubmit = () => {
      // ...
      dispatch(postUserFeedback({ npsScore: 10, content: 'test' }));
  };
```

然後底下的 `mapStateToProps`, `mapDispatchToProps` 和 `hoc` 就不需要了